cvat/apps/engine/models.py

```diff
@@ -325,6 +325,18 @@ class Meta:
     def touch(self) -> None:
         self.save(update_fields=["updated_date"])
 
+@transaction.atomic(savepoint=False)
+def clear_annotations_in_jobs(job_ids):
+    for job_ids_chunk in chunked_list(job_ids, chunk_size=1000):
```

Comment:
Did you compare performance with different `chunk_size`? Why are you using 1000 here and 10000 in the `dataset_manager.task.py::JobAnnotation::_delete`?

Reply:
I did not really compare performance between different chunks, just tried to find something more or less logical and avoid huge SQL requests. Actually I do not think performance between one request with 10000 INs significantly differs with two requests and 5000 INs each (probably the first one will be even quicker). 

In case with annotations, user can easily update > 10000 objects in one save request (e.g. when running an annotations action), splitting it by batches in 100 or 1000 objects, would not be efficient IMHO.  At the same time we can't keep the batch unlimited as the SQL request in general is limited in length for different database backends. 

I will test, btw how it works with many thousands objects .

A task with more than 1000 jobs is something extremely rare on practice and 1000 will be enough almost always to finish operation in one request. Why not set 10000 here? I do not have strong answer here, it just does not look necessary

Reply:
Decreased both to 1000.